code/__DEFINES/~skyrat_defines/layers.dm

```diff
@@ -0,0 +1,5 @@
+///Plane of the "splash" icon used that shows on the lobby screen. Nothing should ever be above this. (Except the logo)
+#define BLACK_FADE 9997
+#define SKYRAT_LOGO 9999
```

Comment:
We should likely fix this, too, while we're thinking about this. Not in this PR, but it's still something that's been broke since you added your HTML menu.

Reply:
For that matter, the lobby_cam module should also be removed since it's just burning CPU and net traffic doing something nobody can ever see anymore.